lib/aspects/resource_prefixer/resource_prefixer.ts

```diff
@@ -27,6 +33,9 @@ export class ResourcePrefixer implements IAspect {
     this.registerPrefixer(CfnTable, DynamoDbCfnTablePrefixer);
     this.registerPrefixer(CfnApi, Apigatewayv2CfnApiPrefixer);
     this.registerPrefixer(CfnStage, Apigatewayv2CfnStagePrefixer);
+    this.registerPrefixer(CfnSecurityGroup, Ec2CfnSecurityGroupPrefixer);
+    this.registerPrefixer(CfnRole, IamCfnRolePrefixer);
+    this.registerPrefixer(CfnFunction, LambdaCfnFunctionPrefixer);
```

Comment:
IMO we should also add the node type on line 57, something along the lines of (the generic error can leave you trying to guess which prefixer is required):
 
```
"Undefined resource for resource prefixer: " + node.cfnResourceType
```

Reply:
I've added this.